addon/templates/components/columns/base.hbs

```diff
@@ -2,8 +2,8 @@
   {{component column.component column=column table=table tableActions=tableActions sortIcons=sortIcons}}
 {{else}}
   {{label}}
-  {{#if column.sorted}}
-    <i class="lt-sort-icon {{if column.ascending sortIcons.iconAscending sortIcons.iconDescending}}"></i>
+  {{#if column.sortable}}
+    <i class="lt-sort-icon {{get sortIcons sortIcon}}"></i>
```

Comment:
👍 This all looks awesome!  I left a note about it in the issue, but I believe this implementation would still show:

```
<i class="lt-sort-icon"></i>
```

in the case that the user has set a column as sortable, but doesn't explicitly pass an `iconSortable` into their `t.head`.  Would we also need some sort of additional check to see if `iconSortable`/`iconAscending`/`iconDescending` is non-blank before showing the icon?  Or even just for `iconSortable`?  That way, the user has the option to not pass one in if they don't want to use it.  It may be ok with `<i class="lt-sort-icon">` showing for `iconAscending` or `iconDescending`, because those may be viewed as required fields to the component interface when dealing with columns that are `sortable`.